HELM-326: Add documentation for location() template_variable (#415)
OpenNMS/opennms-helm: docs/modules/datasources/pages/entity_datasource.adoc

=== alarms([attribute])

The `alarms()` function queries the OpenNMS alarms REST API.

=== locations()

The `locations()` query returns a list of all available node locations. 

> Use this variable with `nodeFilter()` to reduce the number of nodes displayed by location.

Reviewer: ```suggestion
Use this inside `nodeFilter()` to filter the number of nodes displayed by location.
```

> Examples: 
> ```
> nodeFilter(location='Default')
> nodeFilter(location='$location')
> ```

Reviewer: The triple backtick notation for code blocks isn't used by the documentation system.  This should be a list instead.
```suggestion
.Example variable expression
* nodeFilter(location='Default')
* nodeFilter(location='$location')
```

=== nodeFilter(<filterQuery>)

The `nodeFilter()` query returns a list of nodes which match the given filter expression.

The filter expression must be an existing node attribute and its corresponding value. 

> The value can be a constant or a template_variable.

Reviewer: Grafana just calls them "variables" now instead of template variables.
```suggestion
The value can be a constant or a variable.
```

The `alarms()` function will query the OpenNMS alarms ReST API.

=== locations()

The `locations()` query returns a list of all available node locations. 

> Use this variable with nodeFilter to reduce the number of nodes displayed by location.

Reviewer: ```suggestion
Use this variable with `nodeFilter()` to reduce the number of nodes displayed by location.
```

> Use this variable with nodeFilter to reduce the number of nodes displayed by location.

Reviewer 2: ```suggestion
Use this variable with `nodeFilter` to reduce the number of nodes displayed by location.
```

> Examples: +
> ```
> nodeFilter(location='Default')
> nodeFilter(location='$location')
> ```

Reviewer: ```suggestion
Examples:

* nodeFilter(location='Default')
* nodeFilter(location='$location')
```

=== nodeFilter(<filterQuery>)

> The `nodeFilter()` query will return a list of nodes which match the given filter expression.

Reviewer 2: ```suggestion
The `nodeFilter()` query returns a list of nodes which match the given filter expression.
```

> The filter expression must be an existing node attribute and their corresponding value and the value can be constant or a template_variable.

Reviewer: The wording on this seems a little confusing (too many "and/or") for a new user.  Not sure a suggestion yet - need to re-read this section.

> Example: +

Reviewer 2: ```suggestion
Example: 
```

> Example: +
> ```
> nodeFilter(location='Default')
> ```

Reviewer: ```suggestion
Example:

```
* nodeFilter(location='Default')
```
